custom/icds_reports/reports/institutional_deliveries_sector.py

```diff
@@ -138,7 +138,7 @@ def get_data_for(filters):
     }
 
 
-@icds_quickcache(['domain', 'config', 'loc_level', 'show_test'], timeout=30 * 60)
+# @icds_quickcache(['domain', 'config', 'loc_level', 'show_test'], timeout=30 * 60)
```

Comment:
I don't think you meant to commit this, did you?

Reply:
sorry ..it was not intentional.